server/src/main/java/io/unitycatalog/server/utils/JsonUtils.java

```diff
@@ -48,8 +52,10 @@ public static void printJsonAsTable(String jsonString) {
             }
 
         } catch (JsonMappingException e) {
+            LOGGER.error("printJsonAsTable happen JsonMappingException,jsonString:{}",jsonString,e);
             System.out.println(jsonString);
```

Comment:
Can we remove the system out if we add the log?

Reply:
I think it's no problem